docs/articles/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title: Connect to the TestCafe Server over HTTPS
+permalink: /documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html
+---
+# Connect to TestCafe Server over HTTPS
+
+TestCafe is a proxy-based testing tool. Browser requests are sent via the TestCafe proxy server to the tested website. All requests between the browser and the TestCafe server are sent over the HTTP protocol.
+
+![Connection Protocols](../../../images/proxy-connection-protocols.svg)
+
+Some browser features (like
+[Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API),
+[Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API),
+[ApplePaySession](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession), or
+[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto))
+require secure origin. This means that the website should be served over the HTTPS protocol. If you run tests against pages with this kind of browser API in a regular way, these tests fail with JavaScript errors.
+
+TestCafe allows you to connect to the proxy server over the HTTPS protocol to support secure origin testing.
```

Comment:
`to support secure origin testing`  - I don't know that the term `secure origin testing` means.
More strictly : `to support testing web pages using browser features that requires a secure origin`